avaframe/com1DFA/com1DFA.py

```diff
@@ -813,13 +869,15 @@ def initializeParticles(cfg, releaseLine, dem, logName='', relThField=''):
     particles['kineticEne'] = kineticEne
     particles['potentialEne'] = np.sum(gravAcc * mPartArray * particles['z'])
     particles['peakKinEne'] = kineticEne
+    particles['peakForceSPH'] = 0.0
+    particles['forceSPHIni'] = 0.0
     particles['peakMassFlowing'] = 0
     particles['simName'] = logName
     particles['xllcenter'] = dem['originOri']['xllcenter']
     particles['yllcenter'] = dem['originOri']['yllcenter']
 
     # remove particles that might lay outside of the release polygon
-    if not cfg.getboolean('initialiseParticlesFromFile'):
+    if not cfg.getboolean('iniStep') and not cfg.getboolean('initialiseParticlesFromFile'):
```

Comment:
see comment above about potential checking in advance

Reply:
this is required here as both options are possible and if one of them is chosen - particles should not be removed at this location